docs/platform-services/automation-service/app-central/integrations/microsoft-teams.md

```diff
@@ -180,202 +163,163 @@ One of the following permissions is required to call this Action.
 
 One of the following permissions is required to call this Action.
 
-
-
 | PERMISSION TYPE | PERMISSIONS (FROM LEAST TO MOST PRIVILEGED) |
-| --- | --- |
+|:-- | :-- |
 | Delegated (work or school account) | Team.Create |
 | Application | Team.Create, Teamwork.Migrate.All |
 
 **Create Team On Incident Creation Trigger Action:**
 
 One of the following permissions is required to call this Action.
 
-
-
 | PERMISSION TYPE | PERMISSIONS (FROM LEAST TO MOST PRIVILEGED) |
-| --- | --- |
+|:-- | :-- |
 | Delegated (work or school account) | TeamMember.ReadWrite.All |
 | Application | TeamMember.ReadWrite.All |
 
 One of the following permissions is required to call this Action.
 
-
-
 | PERMISSION TYPE | PERMISSIONS (FROM LEAST TO MOST PRIVILEGED) |
-| --- | --- |
+|:-- | :-- |
 | Delegated (work or school account) | Team.Create |
 | Application | Team.Create, Teamwork.Migrate.All |
 
- One of the following permissions is required to call this Action.
-
-
+One of the following permissions is required to call this Action.
 
-| PERMISSION TYPE | PERMISSIONS (FROM LEAST TO MOST PRIVILEGED) |
-| --- | --- |
+| PERMISSION TYPE | PERMISSIONS (FROM LEAST TO MOST PRIVILEGED)|
+|:-- | :-- |
 | Delegated (work or school account) | User.ReadBasic.All, User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
 | Application | User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
 
 **Delete Channel Action:**
 
 One of the following permissions is required to call this Action.
 
```

Comment:
@salvatoremosca, your edits to the tables in the Microsoft Teams article didn't make the tables all the same width. I verified it in a local build.

So I changed the tables back to the way they were so that they don't use spaces to try to make them all the same width (see my commit "[Clean up table format in Microsoft Teams article](https://github.com/SumoLogic/sumologic-documentation/pull/3784/commits/218d927dded657bb1c368e4716f8378503001e84)"). This will make it much easier to maintain them going forward. Now they follow the [standard for tables in our style guide](https://help.sumologic.com/docs/contributing/style-guide/#tables). 

@kimsauce, is there a way to set the width of table columns in our markdown for Docusaurus?